config/labels-annotations.js

```diff
@@ -157,5 +157,7 @@ export const CLUSTER_BADGE = {
   // Badge color - as a hex color - e.g. #ff00ff
   COLOR:       'ui.rancher/badge-color',
   // Use as icon - if set to "true" the cluster icon will also reflect the badge
-  ICON_TEXT: 'ui.rancher/badge-use-as-icon',
+  USE_AS_ICON: 'ui.rancher/badge-use-as-icon',
```

Comment:
the `USE_AS_ICON` annotation should go - we infer this from the presence of the `ICON_TEXT` annotation (when it has 1 or 2 chars)